templates/teams/manage_team.html

```
@@ -17,7 +17,12 @@
         <h3 data-cy="title" class="pg-subtitle">
           {% translate "Team Details" %}
         </h3>
-        <a href="{% url "single_team:feature_flags" request.team.slug %}" class="btn btn-primary">Manage Feature Flags</a>
+        <div class="flex gap-2">
+          {% if request.user.is_staff %}
+            <a href="{% url "single_team:internal_metadata" request.team.slug %}" class="btn btn-primary">Internal Metadata</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)